src/controllers/portfolio/portfolio.ts

```diff
@@ -300,6 +306,16 @@ export class PortfolioController extends EventEmitter {
     }
   }
 
+  async setShouldShowConfetti(accountId: AccountId) {
+    this.#shouldShowConfettiModal = !this.#shouldShowConfettiModal
+    this.shouldShowConfettiBanner = {
+      ...this.shouldShowConfettiBanner,
+      [accountId]: false
+    }
+    await this.#storage.set('shouldShowConfettiBanner', this.shouldShowConfettiBanner)
```

Comment:
In the storage, `shouldShowConfettiBanner` sounds like a boolean. It actually is an object that holds if confetti banner was already shown for a specific account, right? I'd suggest having a more descriptive storage key 🙏